parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -235,7 +235,7 @@ private WriteBuilder createContextFunc(Function<Map<String, String>, Context> ne
             config.put("compression.brotli.quality", compressionLevel);
             break;
           case ZSTD:
-            config.put("io.compression.codec.zstd.level", compressionLevel);
+            config.put("parquet.compression.codec.zstd.level", compressionLevel);
```

Comment:
Should we put both just in case? There isn't backward compatibility?

Reply:
Thanks for your review!
Agreed, both should be put for compatibility with users using older versions of parquet. Will update.

Reply:
Yeah I'd be happy with both as well.